nix/tests/sql/pgbouncer.sql

```diff
@@ -0,0 +1,54 @@
+-- pgbouncer schema owner
+select
+  n.nspname as schema_name,
+  r.rolname as owner
+from
+  pg_namespace n
+join
+  pg_roles r on n.nspowner = r.oid
+where
+  n.nspname = 'pgbouncer';
+
+-- pgbouncer schema functions with owners
+select
+  n.nspname as schema_name,
+  p.proname as function_name,
+  r.rolname as owner
+from
+  pg_proc p
+join
+  pg_namespace n on p.pronamespace = n.oid
+join
+  pg_roles r on p.proowner = r.oid
+where
+  n.nspname = 'pgbouncer'
+order by
+  p.proname;
+
+-- Tests role privileges on the pgbouncer objects
+-- INSERT and UPDATE privileges should not be present on the pgbouncer tables for postgres and service_role, only SELECT and DELETE
```

Comment:
This comment shouldn't be needed, it made sense in the vault test but not here. Better to delete it as it could cause confusion later.
```suggestion
```